frontendActions/src/main/java/com/virtuslab/gitmachete/frontend/actions/base/BaseFastForwardMergeToParentAction.java

```diff
@@ -22,7 +22,7 @@
 import com.virtuslab.logger.IEnhancedLambdaLogger;
 
 @CustomLog
-public abstract class BaseFastForwardParentToMatchBranchAction extends BaseGitMacheteRepositoryReadyAction
+public abstract class BaseFastForwardMergeToParentAction extends BaseGitMacheteRepositoryReadyAction
```

Comment:
Nit: AFAICS from the other classes in the `base` package, `Branch` is included in base class as name (e.g. `BaseResetBranchToRemoteAction`)... anyway, pls make consistent in either way